api/scorer/api.py

```diff
@@ -2,12 +2,24 @@
 The scorer API module
 """
 from ceramic_cache.api import router as ceramic_cache_router
-from django.conf import settings
+from django_ratelimit.exceptions import Ratelimited
 from ninja import NinjaAPI
 from registry.api import analytics_router
 from registry.api import router as registry_router
 
 registry_api = NinjaAPI(urls_namespace="registry", title="Scorer API")
+
+
+@registry_api.exception_handler(Ratelimited)
+def service_unavailable(request, _exc):
+    # pylint: disable=unused-argument,invalid-name
```

Comment:
do we still need to  include the pylint comment if `_exc` is changed to `_` ?

Reply:
Yes, we can drop the pylint comment in that case